manim/camera/moving_camera.py

```diff
@@ -10,41 +10,71 @@
 
 __all__ = ["MovingCamera"]
 
+from typing import Iterable
+
 import numpy as np
 
-from .. import config
-from ..camera.camera import Camera
-from ..constants import DOWN, LEFT, RIGHT, UP
-from ..mobject.frame import ScreenRectangle
-from ..mobject.mobject import Mobject
-from ..utils.color import WHITE
+from manim import config
+from manim.camera.camera import Camera
+from manim.constants import DOWN, LEFT, RIGHT, UP
+from manim.mobject.frame import ScreenRectangle
+from manim.mobject.mobject import Mobject
+from manim.typing import Point3D
```

Comment:
Maybe put this under a ``if TYPE_CHECKING``?

Reply:
Done!